docs/index.rst

```diff
@@ -26,6 +26,9 @@ pages for each source:
 * :doc:`data_sources/ferc1`
 * :doc:`data_sources/ferc714`
 * :doc:`data_sources/phmsagas`
+* :doc:`data_sources/gridpathratoolkit`
+* :doc:`data_sources/nrelatb`
+* :doc:`data_sources/vcerare`
```

Comment:
these other two were also missing from this page so i added. i don't think there is any reason to not have them here